### PR TITLE
docs: remove paragraph in Web Test Runner section

### PR DESCRIPTION
**Pull request by koddsson**
### Description

This PR is addressing possible confusion about Web Test Runner by removing a paragraph from it's section.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [x] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [x] Run the tests with `pnpm test:ci`.

### Documentation
- [x] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: docs/guide/comparisons.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ WebdriverIO comes with the same advantages as Cypress allowing you to test your
 
 [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/) runs tests inside a headless browser, providing the same execution environment as your web application without the need for mocking out browser APIs or the DOM. This also makes it possible to debug inside a real browser using the devtools, although there is no UI shown for stepping through the test, as there is in Cypress tests.
 
-There is a watch mode, but it is not as intelligent as that of Vitest, and may not always re-run the tests you want.
+There is a watch mode, but it is not as intelligent as that of Vitest, and may not always re-run the tests you want.<sup>[[citation needed](https://en.wikipedia.org/wiki/Wikipedia:Citation_needed)]</sup>
```
Comment (AriPerkkio) [inline docs/guide/comparisons.md:45]: This was added in 2021 - maybe this is simply outdated? https://github.com/vitest-dev/vitest/pull/128

I'm afraid if we start adding these `citation needed` labels, the whole docs would eventually fill up with them.

**Comment by 43081j** [comment]
The statement just needs removing, to be honest. 

Not only because it has no explanation, but because it doesn't help the reader in any way. 

As mentioned, it is a pretty old piece of documentation so probably lost its relevance long ago

**Comment by sheremet-va** [comment]
We are not a Wikipedia to add citations because we don't have such guidelines. 

I agree that the sentence should just be removed.

**Comment by koddsson** [comment]
> The statement just needs removing, to be honest.
> We are not a Wikipedia to add citations because we don't have such guidelines.

Sounds good. I was making a bit of a cheeky joke 😄 I can change this PR to just remove the paragraph if that works.

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**